app/backend/src/clj/teet/user/user_db.clj

```diff
@@ -50,3 +51,30 @@
              [(missing? $ ?p :permission/valid-until)]]
            db user-id)
       (mapv first)))
+
+(def user-query-rules
+  '[[(user-by-name ?u ?search)
+     [?u :user/id _]
+     [?u :user/given-name ?given]
+     [(missing? $ ?u :user/deactivated?)]
+     [?u :user/family-name ?family]
+     [(str ?given " " ?family) ?full-name]
+     [(.toLowerCase ?full-name) ?full-name-lower]
+     [(.contains ^String ?full-name-lower ?search)]]])
+
+(defn find-user-by-name [db name]
+  (d/q '[:find (pull ?u [:db/id :user/id :user/given-name :user/family-name :user/email :user/person-id])
+         :where
+         (user-by-name ?u ?name)
+         :in $ % ?name]
+       db
+       user-query-rules
+       (str/lower-case name)))
+
+(defn find-all-users [db]
```

Comment:
The same as the previous - all (d/q ..) functions should use the same style for keys - or put them all in one line, or, all should put some keys to new line as in (user-info) function